packages/components/src/badge/src/Badge.tsx

```diff
@@ -59,7 +59,7 @@ export function InnerBadge({
             <StyleProvider
                 value={{
                     text: {
-                        size: "sm"
+                        size: "xs"
```

Comment:
badge should actually follow a different font, or is this not available in hopper? 
![image](https://github.com/gsoft-inc/wl-orbiter/assets/110498164/c250e0fd-2009-4d49-bf34-297fee5ba9e3)

Reply:
The text transform is not useful for numbers